### DISCUSSION TITLE
Frequent OpenAPI errors

### DISCUSSION THREAD

**Discussion by nommaz** [springbok/MLM2PRO-GSPro-Connector#100] [General]
I used to have a perfectly functioning system which worked really well since v1.04.01 was released for more than one month.

This morning i turned on my system and get very frequent intermittent Failure messages. deleted folder only keeping the config.ini and restored from download to no avail. Also tried to reset connected from GSPro and disconnect / reconnect from the Connector app.

![IMG_4341](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/10116478/d3acce36-41c0-460b-86f5-152d8f2271b9)

Any ideas what else I may need to revert to get my system back?

**Comment by RocknRollGolf** [comment]
Have you checked your ROIs ?

**Comment by nommaz** [comment]
Thanks @RocknRollGolf, I forgot to specify that I am using the bluetooth version (you can see from screenshot). I am not an expert on the connector but I dont think ROI s are involved there.
